shipyard_blueprints/bigquery/shipyard_bigquery/utils/exceptions.py

```diff
@@ -0,0 +1,50 @@
+from shipyard_templates import ExitCodeException
+
+EXIT_CODE_FETCH_ERROR = 101
+EXIT_CODE_QUERY_ERROR = 102
+EXIT_CODE_DOWNLOAD_TO_GCS_ERROR = 103
+EXIT_CODE_TEMP_TABLE_CREATION_ERROR = 104
+EXIT_CODE_SCHEMA_FORMATTING_ERROR = 105
+EXIT_CODE_SCHEMA_VALIDATION_ERROR = 106
+
+
+class InvalidSchema(ExitCodeException):
+    def __init__(self, message: str):
+        self.message = message
+        self.exit_code = 204
```

Comment:
this would be better as 
from shipyard_template import GoogleDatabase

...
self.exit_code= GoogleDatabase.EXIT_CODE_INVALID_SCHEMA